ReadMe.md

```diff
@@ -0,0 +1,88 @@
+# Devoxx Paris 2021 Demo
+
+> This repository demonstrate the usage of Apache Pulsar with a dead simple Node.js application
+
+This demo allows you to a message and analyse the sentiment of the message.
+
+How it works:
+
+- Client sends a request to the server with a message.
+- Server push the message into a Apache Pulsar topic.
+- Worker analyze the message and sends the result into another Apache Pulsar topic.
+- Server reads the result from the analyzed topic and sends them to the client via WebSocket.
+
```

Comment:
Please add articles before nouns. *The client, the server, a worker…*

Please add an *s* to conjugated verbs *pushes*, *analyses*